src/components/TextInputAutoGrow.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import TextInputFocusable from './TextInputFocusable';
+
+const propTypes = {
+
+    // The value of the comment box
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    // A ref to forward to the text input
+    forwardedRef: PropTypes.func.isRequired,
+
+    // General styles to apply to the text input
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.any,
+
+    // Styles to apply to the text input
+    // eslint-disable-next-line react/forbid-prop-types
+    textStyle: PropTypes.any.isRequired,
```

Comment:
Same comment about `PropTypes.any` this shouldn't be `PropTypes.any` because we are not handling the `any` case.